### ISSUE TITLE
Missing Static Class Props

### ISSUE THREAD

**Issue by o0Djeen0o** [r-murphy/babel-plugin-transform-modules-ui5#31]
Hi,

We would like to use [XML Composite](https://sapui5.hana.ondemand.com/#/api/sap.ui.core.XMLComposite%23constructor) in our code, but right now only some static props are handled, especially `metadata` and `renderer`. For `XMLComposite`, we would need in addition at least `fragment`, as shown in the linked constructor.

The easy way would be to add it to the [classes](https://github.com/r-murphy/babel-plugin-transform-modules-ui5/blob/master/packages/plugin/src/classes/helpers/classes.js):
at the two places :
https://github.com/r-murphy/babel-plugin-transform-modules-ui5/blob/b95fbfeb0da8937d79aad50d16eb91cbdddb2cba/packages/plugin/src/classes/helpers/classes.js#L58
https://github.com/r-murphy/babel-plugin-transform-modules-ui5/blob/b95fbfeb0da8937d79aad50d16eb91cbdddb2cba/packages/plugin/src/classes/helpers/classes.js#L118

but maybe being able to configure such behavior would help ?

**Comment by Elberet** [comment]
When a class is thought to be a controller, static properties are handled differently:

https://github.com/r-murphy/babel-plugin-transform-modules-ui5/blob/b95fbfeb0da8937d79aad50d16eb91cbdddb2cba/packages/plugin/src/classes/helpers/classes.js#L35-L36
https://github.com/r-murphy/babel-plugin-transform-modules-ui5/blob/b95fbfeb0da8937d79aad50d16eb91cbdddb2cba/packages/plugin/src/classes/helpers/classes.js#L44-L45
https://github.com/r-murphy/babel-plugin-transform-modules-ui5/blob/b95fbfeb0da8937d79aad50d16eb91cbdddb2cba/packages/plugin/src/classes/helpers/classes.js#L56-L58

In other words, all static properties should be moved to the `extend` clause as long as the class is treated as a controller and the `addControllerStaticPropsToExtend` is enabled. You can force `transform-modules-ui5` to treat your class as a controller by adding a JSDoc tag like this:

```javascript
/** @controller */
export default class Foo ...
```

**Comment by o0Djeen0o** [comment]
@Elberet Thanks, you are right. This is sufficient to add the `@controller` tag.
